### PR TITLE
disasm/exec/validate/wasm: trivial typo fixes

### PR DESCRIPTION
**Pull request by justinclift**
Trivial typo fixes noticed while looking through the code base.

### CODE REVIEW

File: validate/vm.go [hunk 1]
```diff
@@ -23,13 +23,13 @@ type mockVM struct {
...
 
-	polymorphic bool    // whether the base implict block has a polymorphic stack
+	polymorphic bool    // whether the base implicit block has a polymorphic stack
 	blocks      []block // a stack of encountered blocks
 
 	curFunc *wasm.FunctionSig
 }
 
-// a block reprsents an instruction sequence preceeded by a control flow operator
+// a block represents an instruction sequence proceeded by a control flow operator
```
Comment (sbinet) [inline validate/vm.go]: `s/proceeded/preceded/`

Comment (justinclift) [reply validate/vm.go]: Oh good point.  I got that one wrong too. :wink:

Just force pushed an updated commit with that fixed.

**Comment by sbinet** [comment]
needs https://github.com/go-interpreter/license/pull/16.

**Comment by sbinet** [comment]
thanks!

**Pull request merged**